src/templates/doe_workflow_central_ducted_split_heat_pump.mdx

```diff
@@ -0,0 +1,439 @@
+        </PhotoInput>
+        <StringInput label="Notes about thermostat settings" path="comments_about_thermostat_settings" />
+        ## Mechanical Ventilation
+        If the addition of supply based mechanical ventilation is included in the heat pump upgrade. Use the mechanical 
+        ventilation quality installation workflow to document that installation. 
+
+        ## Combustion Safety Testing
+        If an atmospherically vented appliance will remain in the home after this heat pump installation work has been completed,
+        a combustion safety testing has to be performed. Please use the combustion safety documentation workflow to document the combustion safety test results.
+
+    </Tab>
+    <Tab eventKey="report" title="Report">
+        
+
+
+         <PrintSection label="Print Report">
+            ---
+            # Central Ducted Split Heat Pump Upgrade
+            ## Photo Report
+
+            <ProjectInfoReport {...props} />
+        
+            <Photo id="building_number_photo" label="Building Number" required>
+                A photo of the building showing the building number.
+            </Photo>
+            ## About the Report:
+            This report is intended to document:
+            1. The critical tests for evaluating the ductwork in a home to determine what, if any, duct
+            repairs or upgrades are needed so that the upgrade to a heat pump yields the expected
+            efficiency benefits.
+            2. The home’s heating and cooling loads.
+            3. The heat pump equipment selection.
+            4. The heat pump installation.
+            
+            A successful central ducted split heat pump installation starts with well-sealed, insulated, and
+            correctly sized ductwork, and careful equipment planning and selection. Proper installation and
+            commissioning are paramount to ensuring maximum equipment efficiency and a long troublefree equipment life.
+            The photo requirements in this document record the key steps to ensure
+            success. It is strongly recommended a printed copy of this report be attached to the air handler
+            as a record to help future home energy raters and HVAC techs.
+            
+            ## About Atmospherically Vented Water Heaters
+            If there there are atmospherically vented appliances inside the home after the heat pump installation, a combustion safety 
+            test should have been performed to test for backdrafting. Please refer to the combustion safety documentation report if such a test was performed at this house.
+            
+            ## Acronyms
+            ODU – Outdoor Unit <br/>
+            IDU – Outdoor Unit
+           
+            ## Pre-Upgrade Tests
+            
+            <Photo id="existing_duckwork_photo" label="Ductwork – Photo" required>
```

Comment:
Replace "duckwork" with "ductwork".
Check for other cases of this.

Reply:
Corrected the word "duckwork" with "ductwork". Not other occurrences was found.